src/main/resources/jupyter/init-actions.sh

```diff
@@ -247,6 +247,9 @@ if [[ "${ROLE}" == 'Master' ]]; then
       done
     fi
 
+
+    retry 3 docker exec -u root -e PIP_USER=false ${JUPYTER_SERVER_NAME} ${JUPYTER_SCRIPTS}/extension/install_TOC.sh
```

Comment:
Probably should remove the `-u root` -- I think we install extensions as `jupyter-user`.